scripts/docs.js

```diff
@@ -0,0 +1,65 @@
+const StyleDictionary = require('style-dictionary');
+const glob = require("glob");
+const { normalizeTokenName
+} = require("../lib/utils");
+
+
+StyleDictionary.registerTransform({
+    name: "normalizeName",
+    type: "value",
+    transitive: true,
+    matcher:(token)=>token.type==="color",
+    transformer:(token)=>{
+        const {value} = token;
+        return normalizeTokenName(value);
+    }
+    })
+
+// Add a custom format that will generate the tokens related docs format
+StyleDictionary.registerFormat({
+    name: 'json/flatSyncTable',
+    formatter: (dictionary) => {
```

Comment:
please change the formatter to apply `normalizeTokenName` to `prop.path`, like this:

```
formatter: (dictionary) => {
    return (
      "{\n" +
      dictionary.allProperties
        .map(function (prop) {
          return `  "${normalizeTokenName(
            prop.path
          )}": ["${prop.value}","${prop.role}"]`;
        })
        .join(",\n") +
      "\n}"
    );
  },
```

that will generate the desired format, e.g.:

```
"color-ui_grey-muted": ["#ecedf5","natural"],
```